packages/hardhat-chai-matchers/src/types.ts

```diff
@@ -1,5 +1,3 @@
-declare module "deep-eql";
```

Comment:
Oh, this was necessary when `deep-eql` was a top-level import. After we moved it to a lazy require (not an `await import` because it's used inside a sync function), this module declaration became unnecessary.